internal/k8s/service/resolver.go

```diff
@@ -169,31 +169,28 @@ func (r *ResolvedReference) Item() client.Object {
 }
 
 type BackendResolver interface {
-	Resolve(ctx context.Context, ref gw.BackendObjectReference) (*ResolvedReference, error)
+	Resolve(ctx context.Context, namespace string, ref gw.BackendObjectReference) (*ResolvedReference, error)
```

Comment:
Assuming this change is to have the resolver be more of a longstanding global that isn't tied to a single namespace like it was before?

Reply:
yeah, some of the small changes like this were to make it so that these service structures were usable without having to re-initialize them -- so they are no longer scoped to resolving a single route's backends (via its namespace), but instead can be initialized once and called for any route that needs to get resolved.